lib/logstash/outputs/pravega.rb

```diff
@@ -57,8 +60,8 @@ def pre_check(encoded)
     encoded.each do |event, data|
```

Comment:
This part of the old code seems useless and should be removed. It should not be looking at the streamName field of the event.

Reply:
pre_check should be removed.